share/module_check_a_mundo.F

```diff
@@ -1089,6 +1089,24 @@ END FUNCTION bep_bem_nbui_max
 !-----------------------------------------------------------------------
 ! cu_physics = 11 (scale-aware KF)
```

Comment:
Wei,
I think that these "cu_physics = 11" header/comment lines belong with the IF test after the new 3dtke.